ports/libtheora/portfile.cmake

```
@@ -21,12 +21,13 @@ endif()
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
+        -DVERSION:STRING="${VERSION}"
```

```suggestion
        "-DVERSION:STRING=${VERSION}"
```
Quoting for CMake, not for the shell.